src/JuliusSweetland.OptiKey.Core/Models/XmlKeyboardModels/XmlDynamicItem.cs

```diff
@@ -0,0 +1,84 @@
+﻿// Copyright (c) 2019 OPTIKEY LTD (UK company number 11854839) - All Rights Reserved
+using JuliusSweetland.OptiKey.Enums;
+using System.Collections.Generic;
+using System.Xml.Serialization;
+
+namespace JuliusSweetland.OptiKey.Models
+{
+    public class XmlDynamicItem : XmlVisualItem
+    {
+        public XmlDynamicItem() { }
+
+        [XmlElement("Action", typeof(DynamicAction))]
```

Comment:
I'm guessing the concept of a basic XmlKey is being removed and all keys are now XmlDynamicItems which can contain 1 step to replicate the behaviour of the previous style of keys?